### DISCUSSION TITLE
Lasso results are different from LightGBM/RF/Xgboost

### DISCUSSION THREAD

**Discussion by victoriasunsun** [DoubleML/doubleml-for-py#140] [Q&A] [answered]
Hi, the DML package is really useful for me and I am using it to conduct my master thesis. I have tried LightGBM/RF/Xgboost/Lasso for learners. The results of LightGBM/RF/Xgboost are similar but the results of Lasso is rather different. The following is a part of the results. Can you help me with that issue?

indicators|Lasso | lightGBM | Xgboost | RF
--|-- | -- | -- | --
a|-0.105 | -13.424*** | -13.410*** | -13.025***
b|0.001 | 0.265*** | 0.275*** | 0.259***
c|0.003 | 0.186*** | 0.187*** | 0.185***
d|-0.017 | 20.600*** | 21.417*** | 20.165***
e|1.701 | 13.227*** | 13.282*** | 12.853***
f|16.672 | 10.549* | 15.637** | 8.339

**Comment by PhilippBach** [comment, marked as answer]
We moved this discussion to the discussions section of the R package, see https://github.com/DoubleML/doubleml-for-r/discussions/149
